.github/workflows/pixi-testing.yml

```diff
@@ -0,0 +1,29 @@
+name: Unit Tests
```

Comment:
I assume in the future this would replace `testing.yml`?

Reply:
Yep it was to have a first push to not remove the previous file :)